tutorials/chartjs/size.md

```diff
@@ -0,0 +1,118 @@
+To set the chart size in chartjs, you should use the `responsive` option.
+You must wrap the chart canvas tag in a div with the css property `position` set to `relative`.
+By default, the option is enabled and set to true.
+
+<style>
+  #chart-wrapper {
+    display: inline-block; position: relative; height: 400px; width: 66%;
+  }
+</style>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
+<div id="chart-wrapper">
+  <canvas id="chart"></canvas>
+</div>
+<script>
+  const ctx = document.getElementById('chart').getContext('2d');
+  const chart = new Chart(ctx, {
+      // The type of chart we want to create
+      type: 'bar',
+      // Configuration options go here
+      options: {
+        responsive: true, // ← comment this out to see no change
+        scales: {
+          yAxes: [{
+            ticks: {
+              beginAtZero: true
+            }
+          }]
+        }
+      }
+  });
+</script>
+
+```html
+<style>
+  #chart-wrapper {
+    display: inline-block; position: relative; height: 400px; width: 66%;
+  }
+</style>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
+<div id="chart-wrapper">
+  <canvas id="chart"></canvas>
+</div>
+<script>
+  const ctx = document.getElementById('chart').getContext('2d');
+  const chart = new Chart(ctx, {
+      // The type of chart we want to create
+      type: 'bar',
+      // Configuration options go here
+      options: {
+        // responsive: true,
+        scales: {
+          yAxes: [{
+            ticks: {
+              beginAtZero: true
+            }
+          }]
+        }
+      }
+  });
+</script>
+```
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
```

Comment:
Are these subsequent examples necessary?

Reply:
If the user resizes the window, it will show how the setting affects the graph. I could make a comment telling them to resize the window to see.